build.gradle.kts

```diff
@@ -5,6 +5,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
     kotlin("jvm") apply false
     id("com.jfrog.bintray") apply false
+    id("java")
```

Comment:
I don't think wee need to apply it to top level project as it is only an aggregate.

Reply:
I think we do, otherwise we can't configure it in the `subprojects` block, but maybe I'm missing something